api/models/User.js

```diff
@@ -4,23 +4,32 @@ const mongoose = require('mongoose');
 
 // User Schema Definition
 var userSchema = new mongoose.Schema({
-	name: String
+	name: {type: String , required: true},
+	isAdmin: {type: Boolean , default: false},
+	isMentor: {type: Boolean , default: false},
+	skills: [String],
+	googleID: { type: String , unique: true, required: true }
 });
 
 // User Schema Methods
 
 /**
  * Creates and saves a User object in the database
- * @param {string} user's name
+ * @param {string} user's name, {string} user's GoogleID (as hash)
  * @returns {User} newly saved User object
  * @example
- * User.create("Joe Bruin")
+ * User.create('Joe Bruin', 'someGoogleIdHash')
  *     .then(user => console.log(user))
  *     .catch(error => console.error(error));
  */
-userSchema.statics.create = function(name) {
+ 
+var User;
```

Comment:
The declaration of User here seems kinda randomly placed. You should put this like near the top so it’s easier to find.